### ISSUE TITLE
Error when using some Ternary Operators code

### ISSUE THREAD

**Issue by calebeaires** [DotJoshJohnson/vscode-xml#76]
On a Nativescript xml coding, when using Ternary Operators with `<` I got the error above, but not when using something like that `!=`:

`XML formatting failed: at line 13, column 76: expected exclamation mark, question mark or alphabetic tag name.`

```
<StackLayout>
    <Label text="label with just < " visibility="{{ p.status < 1 ? 'visible' : 'collapsed' }}"  />
    <Label text="labe with != " visibility="{{ p.status != 1 ? 'visible' : 'collapsed' }}" />
</StackLayout>
```

**Comment by DotJoshJohnson** [comment]
Per the [RFC](https://www.w3.org/TR/xml/#d0e888) for the XML standard, the less than character cannot be used in an attribute value. It would need to be encoded as `&lt;` first. Because of this, most XML parsers will take issue with your XML.

With the above said, my extension would not throw the error you mentioned as I do not rely on a parser to format the document (it is, for better or worse, done "manually"). If you attempt to format the above XML with the *XML Tools* extension, you'll get the following:

```xml
<StackLayout>
    <Label text="label with just < " visibility="{{ p.status 
        < 1 ? 'visible' : 'collapsed' }}" />
        <Label text="labe with != " visibility="{{ p.status != 1 ? 'visible' : 'collapsed' }}" />
    </StackLayout>
```
Still not pretty, but it is behaving as expected given the less than character is not encoded properly.

In your case, I think you may have another XML extension installed that overrides the format command. The *[XML Formatter](https://marketplace.visualstudio.com/items?itemName=fabianlauer.vs-code-xml-format)* extension is another popular XML extension and I believe it uses a parser to format the document, which could throw this error.

At any rate, I will treat this as an enhancement and evaluate the possibility of introducing a setting that makes the formatter a bit more lenient here. I'm not sure of the effort required, but I promise I'll look into it!

**Comment by DotJoshJohnson** [comment]
This will be included in v2.x.

**Comment by DotJoshJohnson** [comment]
Released in [`v2.0.0-preview.1`](https://github.com/DotJoshJohnson/vscode-xml/releases/tag/v2.0.0-preview.1). This will be released to the marketplace when `v2.0.0` is out of preview.
